nomination/views.py

```diff
@@ -481,6 +481,10 @@ def url_add(request, slug):
                     posted_data = handle_metadata(request, posted_data)
 
                     summary_list = add_url(project, posted_data)
+                    if not summary_list:
+                        return HttpResponse('It looks like you have multiple accounts. Please'
+                                            'contact the project admin for further assistantance.',
+                                            content_type='application/json')
```

Comment:
What is the rationale behind a content_type of 'application/json'? Also, typo: "assistantance".

For the project admin, I think it would make sense to use the [Project admin_email value](https://github.com/unt-libraries/django-nomination/blob/0bf57406aee3d71fae35e0ad791450445328ac4b/nomination/models.py#L129).

Reply:
I am also wondering if rather than saying specifically that the user has multiple accounts, to generically say something that could be used for the other exceptions too. Like "There was a problem processing your nominator details. Please contact ... for assistance and provide the name, email address, and institution you are using for nominations.